송원/오등큰수/index.py

```diff
@@ -0,0 +1,32 @@
+import sys
+sys.stdin = open("input.txt", "rt")
+
+n = int(input())
+sequence = list(map(int, input().split()))
+sequenceSet = set(sequence)
+answer = []
+F = {}
+
+maxCount = 0
+
+for i in sequenceSet:
+  count = sequence.count(i)
+  if count > maxCount:
+    maxCount = count
+  F[i] = count
+
+def NGF(index):
+  if F[sequence[index]] == maxCount:
+    return answer.append(-1)
+
+  for i in range(index + 1, len(sequence)):
+    if F[sequence[index]] < F[sequence[i]]:
+      return answer.append(sequence[i])
+  return answer.append(-1)
+
+for index in range(len(sequence)):
+  NGF(index)
+
+print(' '.join([str(i) for i in answer]))
```

Comment:
```suggestion
print(*answer)
```
[서영님 코드](https://github.com/Yapp-17th-Algorithm/algorithm-python/pull/9/files#diff-de974c52d9bfba9dfb092bbee362b8afR22)를 보면 이렇게 줄일 수 있을 것 같습니다.

해당 문법은 찾아보니 [리스트언패킹](https://dojang.io/mod/page/view.php?id=2345) 이라고 하네요!